docs/index.html

```diff
@@ -178,8 +178,9 @@
 				<div class="text-inner">
 					<div class="row">
 						<div class="col-md-8 col-md-offset-2 text-center">
-							<h1 class="to-animate">Enable your business with <span>Blockchain</span></h1>
-							<h2 class="to-animate">Build the trusted economy in IoT, securing data with IOTA <a href="http://creativecommons.org/licenses/by/3.0/" target="_blank"> </a>
+							<h1 class="to-animate">Start your business with <span>Blockchain Technology</span></h1>
+							<h2 class="to-animate">Build up the reliable IoT system, Securing Data with IOTA
+								<a href="http://creativecommons.org/licenses/by/3.0/" target="_blank"> </a>
```

Comment:
Why would you insert CC link here?

Reply:
I'll check it out to remove the unnecessary link